src/util.js

```diff
@@ -285,8 +285,12 @@ export function isClose(a, b, delta) {
 	return Math.abs(a - b) <= delta;
 }
 
+export function getPointer() {
+	return canvas.app.renderer.plugins.interaction?.mouse ?? canvas.app.renderer.events.pointer
```

Comment:
I really like the introduction of a function for this.

Why is it querying two different paths though? Are there any cases where canvas.app.renderer.plugins.interaction.mouse still works in v11, where canvas.app.renderer.events.pointer doesn't?

Reply:
The first method works under v10 and the second under v11. This function appears to be a shim to provide simultaneous v10 and v11 support.

Reply:
Actually I'm not sure. I haven't checked on V10 and V11. I assumed it would be easier try to get pointer for V10 and if it is absent - take for V11. We do not want to break V10 systems.

Reply:
considering interactionData - not sure as well. Can someone check it on V10?

Reply:
Breaking v10 is fine. The old versions for Drag Ruler stay available for users that opt not to move over to v11, so using Drag Ruler will stay viable in any case - and just marking v10 as incompatible will save us from doing tons of testing on different versions.